### ISSUE TITLE
AJAX option not work

### ISSUE THREAD

**Issue by yammycode** [bobdenotter/forms#22]
It doesn't matter if the option is set or not - the page reloads.

**Comment by jlgranda** [comment]
how this works?

**Comment by aj247** [comment]
Confirmed not working in latest version. Looks like it hasn't been implemented yet?

**Comment by I-Valchev** [comment]
@aj247 yes, it looks like no one has gone around to implement it, _yet_.

PRs are welcome!

**Comment by binarykiwi** [comment]
Is there any update on this?
